book/tut4_geophysics.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
_Commented on notebook `book/tut4_geophysics.ipynb` Cell 37 Line 1_
```python
pygmt.makecpt(cmap="acton", series=[0, 500], reverse=True)
```
TEST: Use another colormap, as the SC colormap "lajolla" flips between the different (GMT) versions.

Reply:
Changed in commit https://github.com/GenericMappingTools/agu24workshop/pull/8/commits/334ef23b3791a366f59de3ee698034344dbe58f2.